config/host.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// host.js
+//module.exports.host = 'stream.watsonplatform.net';
+module.exports.host = 'stream-s.watsonplatform.net';
```

Comment:
I know that you need this for your copy of the demo right now, but this should be set to the public URL before I merge it.

Ideally, I'd like to see this file removed and not even imply that the host is configurable, since it just adds extra overhead for most users.